apps/web/src/components/(dashboard)/layout/header.tsx

```diff
@@ -33,13 +33,13 @@ export const Header = ({ className, user, ...props }: HeaderProps) => {
   return (
     <header
       className={cn(
-        'supports-backdrop-blur:bg-background/60 bg-background/95 sticky top-0 z-[50] flex h-16 w-full items-center border-b border-b-transparent backdrop-blur duration-200',
+        'supports-backdrop-blur:bg-background/60 bg-background/95 sticky top-0 z-[50] flex h-16 w-[100vw] items-center border-b border-b-transparent backdrop-blur duration-200 lg:w-[calc(100vw-2rem)] lg:pl-4',
```

Comment:
This seems like it'll potentially cause a horizontal scrollbar due to the fixed sizing, the `lg` breakpoint class also feels like a hack which would be best to avoid 😬